app/src/main/java/nextstep/shoppingcart/ui/component/CartCountController.kt

```diff
@@ -0,0 +1,55 @@
+package nextstep.shoppingcart.ui.component
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+@Composable
+internal fun CartCountController(
+    count: Int,
+    onPlusClick: () -> Unit,
+    onMinusClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Row(
+        horizontalArrangement = Arrangement.spacedBy(12.dp),
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = modifier
+    ) {
+        IconButton(
+            onClick = onMinusClick,
+            modifier = Modifier.testTag("장바구니::아이템수량감소")
+        ) {
+            Text(text = "-", style = MaterialTheme.typography.headlineMedium)
+        }
+        Text(
+            text = count.toString(),
+            style = MaterialTheme.typography.headlineSmall,
+            modifier = Modifier.testTag("장바구니::아이템수량")
+        )
+        IconButton(
+            onClick = onPlusClick,
+            modifier = Modifier.testTag("장바구니::아이템수량증가")
+        ) {
+            Text(text = "+", style = MaterialTheme.typography.headlineMedium)
+        }
+    }
+}
+
+@Preview(showBackground = true)
+@Composable
+private fun CartCountControllerPreview() {
+    CartCountController(
+        count = 4,
+        onPlusClick = {},
+        onMinusClick = {}
+    )
+}
```

Comment:
`CartCountController`의 동작을 검증하기 위해 다음과 같이 작성하면 Preview의 Interactive Mode를 활용하여 직접 Count가 어떻게 증감되는지 확인할 수 있습니다.
```suggestion
@Preview(showBackground = true)
@Composable
private fun CartCountControllerPreview() {
    var count by remember { mutableIntStateOf(4) }

    CartCountController(
        count = count,
        onPlusClick = { count++ },
        onMinusClick = { count-- }
    )
}
```